pages/classes/wise.md

```diff
@@ -159,17 +169,13 @@ You channel a burst of raw, primordial energy through your body. You or a [Close
 
 </header>
 
-You can use your [Action](../../pages/combat/actions.md) to assume an animal form. You use this feature again when you [Rest](../../pages/rules/rests.md).
-
-When you transform you have **10 + your level Stamina**. When you revert this Stamina is lost, and you return to your Stamina before you changed.
+When you use your **Animal Form** feature you gain an additional **1d4 + 4 Stamina** in your new form.
 
-When you transform you may pick 2 of the following benefits for your new form:
+When you transform you can choose one of the following for your new form:
 
- * You gain a **Bite or Claw (Melee 1d6, Light) Attack**.
- * You become tiny and/or can fly (you cannot attack in this form).
- * You can breath water.
- * You can use a [Bonus Action](../../pages/combat/bonus-actions.md) to make the [Dash](../../pages/combat/actions.md#dash) [Action](../../pages/combat/actions.md).
- * You have advantage on all [Perception](../../pages/characters/skills.md#perception/) [Checks](../../pages/rules/rolling/checks.md).
+ * You add your **Wisdom** to your attack damage
+ * You can see in darkness
+ * You become proficient in **Athletics** or **Stealth**
```

Comment:
This is supposed to be equivalent to the Beguiling Influence eldritch invocation, but may be over-powered because Athletics and Stealth have combat utility.